packages/fetchai/skills/generic_seller/generic_data_model.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains the dataModel for the generic seller aea."""
+
+from aea.helpers.search.models import Attribute, DataModel
+
+SCHEME = {"country": "UK", "city": "Cambridge"}
```

Comment:
Nothing about this data model is generic. It's attributes and names should be specified in the configuration, not hard coded. To achieve this I'd create a class `class SpecificDataModel(DataModel, SharedClass):`... so you can configure it from the skill yaml